packages/compass-shell/src/components/compass-shell/compass-shell.tsx

```diff
@@ -304,9 +304,7 @@ export class CompassShell extends Component<
 
 export default connect((state: RootState) => ({
   emitShellPluginOpened: () => {
-    if (state.appRegistry && state.appRegistry.globalAppRegistry) {
-      state.appRegistry.globalAppRegistry.emit('compass:compass-shell:opened');
-    }
+    state.runtime.appRegistry?.emit('compass:compass-shell:opened');
```

Comment:
Is this used anywhere? Maybe we can remove it 🤔

Reply:
Only in a test … I’m guessing it’s fine to just remove that entire test?

Reply:
(Doh, I had test files excluded in vscode search and didn't notice the test!) I'd clean it up, yeah, the less unused events we have, the easier it will be to assert their usage. But no strong preference really, feel free to keep it